### PR TITLE
Added required text for required elements on a Webform. Fixes #719.

### LINKED ISSUE #710: “required” isn’t displaying next to required checkbox and radio button options on forms

### LINKED ISSUE #719: “Required” isn’t displaying next to required checkbox and radio button options on forms
> I don’t know if anyone reported this one yet, sorry if it’s a repeat. It looks like “required” isn’t displaying next to required checkbox and radio button options on forms https://cms.unl.edu/executive-vice-chancellor/node/477/

### PR DESCRIPTION
**Pull request by sarongebre**
Added required text to display if an element in a Webform is required for fieldset elements. Closes #719.
Closes #710

### CODE REVIEW

File: web/modules/custom/unl_webform/unl_webform.module [hunk 1]
```diff
@@ -57,42 +57,40 @@ function unl_webform_form_alter(&$form, FormStateInterface &$form_state, $form_i
...
-            $remove_group = FALSE;
-          }
-        }
-        if ($remove_group) {
-          unset($form[$key]);
-        }
-      }
-    }
-  }
-}
```
Comment (ericras) [inline web/modules/custom/unl_webform/unl_webform.module:95]: Why is this all removed?

File: web/modules/custom/unl_webform/unl_webform.module [hunk 2]
```diff
@@ -106,8 +104,12 @@ function unl_webform_form_webform_ui_element_form_alter(&$form, FormStateInterfa
   $element_type = $element['#type'];
   // Add element type to $form array to make it available to
   // validate function(s).
+  if ($element_type == 'managed_file'
+    || $element_type == 'webform_document_file'
+    || $element_type == 'webform_image_file'
+    ) {
   $form['element_type'] = $element_type;
-
+    }
```
Comment (ericras) [inline web/modules/custom/unl_webform/unl_webform.module:112]: Why is this "if" statement added?

File: web/modules/custom/unl_webform/unl_webform.module (hunk 1 above)
Comment (sarongebre) [reply web/modules/custom/unl_webform/unl_webform.module]: Sorry, I think it got removed by accident while I was configuring and experimenting with some VS Code formatting plugins. I have added it back in.

File: web/modules/custom/unl_webform/unl_webform.module (hunk 2 above)
Comment (sarongebre) [reply web/modules/custom/unl_webform/unl_webform.module]: Sorry, I think it got added by accident while I was configuring and experimenting with some VS Code formatting plugins. I have removed it.

**Pull request merged**